pkg/diff/diff.go

```diff
@@ -258,6 +258,7 @@ func removeWebhookMutation(predictedLive, live *unstructured.Unstructured, gvkPa
 	}
 
 	if comparison.Modified != nil && !comparison.Modified.Empty() {
```

Comment:
Is this bug also potentially an issue for added fields? If not, why not? Maybe should add a comment to explain.

Reply:
We don't call merge, so maybe not, but I'll check.